client/packages/system/src/Stock/Components/Repack/RepackEditForm.tsx

```diff
@@ -117,24 +117,33 @@ export const RepackEditForm: FC<RepackEditFormProps> = ({
               />
             }
           />
-          <InputWithLabelRow
-            label={t('label.new-location')}
-            labelWidth="100%"
-            Input={
-              <LocationSearchInput
-                autoFocus={false}
-                disabled={!isNew}
-                selectedLocation={location}
-                width={160}
-                onChange={location => {
-                  setLocation(location);
-                  onChange({
-                    newLocationId: location?.id,
-                  });
-                }}
-              />
-            }
-          />
+          {data.newLocationName ? (
```

Comment:
If new location was not set, we still show not editable drop down ?

Reply:
It shows as { code } - { name } and would need tooltip on hover... which was different from how we were showing old location...